MoltenVK/MoltenVK/Utility/MVKLogging.h

```diff
@@ -28,16 +28,15 @@ extern "C" {
 #include <stdio.h>
 #include <string.h>
 #include <assert.h>
-#include <asl.h>
 #include <stdbool.h>
 
 /**
  * This library adds flexible, non-intrusive logging and assertion capabilities
  * that can be efficiently enabled or disabled via compiler switches.
  *
  * There are five levels of logging: Trace, Info, Warning, Error and Debug, and each can be enabled
- * independently via the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_INFO, MVK_LOG_LEVEL_WARNING,
- * MVK_LOG_LEVEL_ERROR and MVK_LOG_LEVEL_DEBUG switches, respectively.
+ * independently via the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_DEBUG, MVK_LOG_LEVEL_INFO,
+ * MVK_LOG_LEVEL_WARNING анд MVK_LOG_LEVEL_ERROR switches, respectively.
```

Comment:
Clean up the list of five logging levels and how they map to `MVK_LOG_LEVEL_*`s. Also...is that Cyrillic text (or something) in the "and"?

Reply:
> Clean up the list of five logging levels and how they map to MVK_LOG_LEVEL_*s.

What do you mean, remove these lines from the description? Or reorder ```Trace, Info, Warning, Error and Debug``` line?

> ...Also...is that Cyrillic text (or something) in the "and"?

Oh yes, I'm sorry, I didn't notice.

Reply:
>Or reorder Trace, Info, Warning, Error and Debug line?

Yes. The "respectively" indicates the reader should expect the same ordering between the `Trace, Info...` list and and "MVK_LOG_LEVEL..." list.

Reply:
But frankly...I'm also fine if you remove the first list, and just have:

```
There are five levels of logging, controlled by the MVK_LOG_LEVEL_TRACE, MVK_LOG_LEVEL_DEBUG,
MVK_LOG_LEVEL_INFO, MVK_LOG_LEVEL_WARNING аnd MVK_LOG_LEVEL_ERROR switches.
```

Reply:
Fixed. 0da221f4